### DISCUSSION TITLE
ReNet Inference and Data Generation

### DISCUSSION THREAD

**Discussion by arunksagotra** [pyg-team/pytorch_geometric#8534] [General]
Hi,

I would like to know how we can run inference for ReNet model. I didn't understand perfectly how are generating sequences. For e.g. if your sequence length is 10 means you are considering last 10 days of event or only just 10 events?

Thanks

**Comment by rusty1s** [comment]
It's been some time since I last looked at `ReNet`. `seq_len` would denote the number of events (independent on the number of days).
